### DISCUSSION TITLE
Is there a way to "downgrade" the version of Sage?

### DISCUSSION THREAD

**Discussion by graywolfusmc** [OpenSageTV/Community#78] [General]
Got a very weird issue. Have 3 Sage servers, 2 are at 9.2.1.826 and 1 is at 9.2.6.976
The ones at  9.2.1.826 are working fine, it is the 9.2.6.976 version that has the weird issue.
INSP (DirecTV 364), on the Guide it shows Gunsmoke at 10pm (ET), Midnight (ET), and 7am (ET) on all 3 machines. 
If I go to Future Airings for  Gunsmoke I see the Midnight Airing on the 9.2.1.826 servers but not on the 9.2.6.976 server.
And it is not just the Midnight Airing, but also the 10pm & 7am episodes.

This happens with the stock Sage7 UI and with Gemstone.
Attached are the screenshots showing the issues

Any thoughts, advice, comments?

![9 2 1-Additional-Airings](https://github.com/OpenSageTV/Community/assets/3332485/6a47bb7d-118a-4e0c-8a98-e08fd9ecace5)
![9 2 6-Additional-Airings](https://github.com/OpenSageTV/Community/assets/3332485/69ac0dfe-9696-40b7-96e1-f254c6373bf3)

**Comment by emveepee** [comment]
Assuming you are using Schedules Direct are you sure your EPG update is working every night on all your server instances?    Recently Schedules Direct had to make some changes to adapt to SageTV and some EPG updates would have failed on multiple server environments.  

Best to make sure the EPG update times are not concurrent.

**Reply by graywolfusmc** [reply]
Yes. Using Schedules Direct. I ensure that they do not run EPG update at same time since I have batch files that control the start of the EPG updates. 

Also, as I said, when looking via the Guide, all 3 show the 10pm (ET), Midnight (ET), and 7am (ET)  airings. It is just the Future/Additional Airings screen where it is not showing..

**Reply by emveepee** [reply]
Understood since I don't use SageTV I don't know how the errors that occurred earlier this week would have impacted linkages.  The only way you could have avoided errors might have been restarting servers in your scripts, it was more than an overlap issue.

**Comment by LoisGNS** [comment]
So each server sees the same group of episodes in the Guide, but when you check Future Airings, one server is missing episodes showing in the others? If that's the case, I would look into metadata as the problem: the episodes are humanly-recognizable as belonging to the same show, but maybe there's some metadata difference between the ones that are found in Future Airings and the ones that aren't, and maybe different servers are interpreting it differently. These are pure guesses; I don't know my way around the source code **at all**.

Is it only one show that shows this problem or have you checked several? I would compare the metadata on the missing shows vs. the ones that show up. Maybe some of the shows aren't recognized as being the "same" show as the others. I'm not sure how much metadata you can see, though, on future airings vs. actual recordings, but maybe there's a hidden character somewhere. I'd also walk through as many settings on the "bad" vs "good" servers to see if any potentially-relevant differences jump out. 

I think I've also had searches fail to show recordings that I know are present, so I suspect the search engine is somewhat buggy in general.
